src/databases/CGNS/avtCGNSFileReader.C

```diff
@@ -1777,350 +1782,1018 @@ avtCGNSFileReader::GetUnstructuredMesh(int timestate, int base, int zone, const
+//
+//      Read mixed and named element sections, and add the cells from
+//      these sections to a given unstructured grid.
+//
+//      NOTE: the contents of this method were largely migrated from
+//      GetUnstructuredMesh, which was originally authored by Brad Whitlock.
+//
+//  Arguments:
+//      urgrid                  The unstructured grid to add our cells to.
+//      meshName                The name of our mesh.
+//      mixedAndNamedSections   An array of our mixed and named section indices.
+//      base                    The CGNS base to use.
+//      zone                    The CGNS zone (domain) to use.
+//      cellDim                 Dimension of the cells; 3 for volume cells,
+//                              2 for surface cells and 1 for line cells.
+//      physDim                 Number of coordinates required to define a
+//                              vector in the field.
+//
+//  Programmer: Alister Maguire
+//  Creation:   Tue Feb  2 09:20:04 PST 2021
+//
+//  Modifications:
+//
+// ****************************************************************************
+
+void
+avtCGNSFileReader::ReadMixedAndNamedElementSections(vtkUnstructuredGrid *ugrid,
+    const char *meshName, std::vector<int> &mixedAndNamedSections, int base,
+    int zone, int cellDim, int physDim)
+{
+    bool higherOrderWarning = false;
+    const char *mName = "avtCGNSFileReader::ReadMixedAndNamedElementSections: ";
+
+    //
+    // Iterate over the mixed and named sections, and add each zone to
+    // our mesh.
+    //
+    for(std::vector<int>::iterator secItr = mixedAndNamedSections.begin();
+        secItr != mixedAndNamedSections.end(); ++secItr)
+    {
+        char sectionName[33];
+        ElementType_t secElemType    = ElementTypeNull;
+        int sec                      = *secItr;
+        cgsize_t start               = 1;
+        cgsize_t end                 = 1;
+        cgsize_t elementSizeInterior = 0;
+        int bound                    = 0;
+        int parentFlag               = 0;
+        int elem_status              = CG_OK;
+
+        if (cg_section_read(GetFileHandle(), base, zone, sec, sectionName,
+            &secElemType, &start, &end, &bound, &parentFlag) != CG_OK)
+        {
+            debug1 << mName << cg_get_error() << endl;
+            continue;
+        }
+
+        if (parentFlag > 0)
+        {
+            debug4 << mName << "parentFlag = " << parentFlag << endl;
+            continue;
+        }
+
+        elementSizeInterior = (end - start + 1);
+
+        if (cellDim == physDim)
+        {
+            elementSizeInterior -= bound;
+        }
+
+        cgsize_t eDataSize = 0;
+
+        if (cg_ElementDataSize(GetFileHandle(), base, zone, sec, &eDataSize)
+            != CG_OK)
+        {
+            debug1 << mName << "Could not determine ElementDataSize\n";
+            continue;
+        }
+
+        debug4 << "Element data size for sec " << sec << " is:"
+            << eDataSize << endl;
```

Comment:
Align the continuation line on the "<<".

Reply:
Done.